code/lib/ui/package.json

```diff
@@ -66,6 +66,8 @@
     "@storybook/semver": "^7.3.2",
     "@storybook/theming": "7.0.0-alpha.34",
     "@testing-library/react": "^11.2.2",
+    "@types/node": "^14.0.10 || ^16.0.0",
+    "@types/webpack-env": "^1.16.0",
```

Comment:
Don't we build these with esbuild, not webpack? Is it weird to depend on webpack env types?

Reply:
module.hot.accept does not exist in node, that's a webpack runtime thing

Reply:
We use that in `lib/ui`???